test/e2e/arktos/multi_tenancy/test_suites/kubectl/test_cp_exec.yaml

```diff
@@ -0,0 +1,66 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Kubectl cp & kubectl exec Tests ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This test suite verifies command "kubectl cp" and "kubectl exec"
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+######################################################
+# test variables
+######################################################
+Variables:
+  test_tenant: random_8
+  test_ns: random_8
+
+###########################################################################################################
+# test setup
+###########################################################################################################
+Tests:
+  - BeforeTestMessage: Starting test setup ...
+    Command: ${kubectl} create tenant ${test_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${test_tenant} created\n"
+
+  - Command: ${kubectl} create namespace ${test_ns} --tenant ${test_tenant}
+    OutputShouldBe: "namespace/${test_ns} created\n"
```

Comment:
\n at the end may not be needed

Reply:
Actually, "\n" at the end is necessary. The comparison is based on "OutputShouldBe", which is an exact match. The test will fail without the ending "\n".